src/api/user/model.js

```diff
@@ -32,8 +32,9 @@ const UserSchema = new Schema({
   imageUrl: String,
 });
 
-UserSchema.virtual('name').get(function get() {
-  return `${this.firstName} ${this.lastName}`;
+UserSchema.virtual('displayName').get(function get() {
+  const postfix = this.lastName ? ` ${this.lastName}` : '';
+  return `${this.firstName}${postfix}`;
```

Comment:
`${this.firstName}${this.lastName || ''}`;

Reply:
That does not equal to what is written. I need a space before `lastName` if it is presented.